src/mvi/gpgmm_d3d12.cpp

```diff
@@ -384,6 +378,9 @@ namespace gpgmm::d3d12 {
         *ppResourceAllocationOut = new ResourceAllocation(
             allocationDesc, this, static_cast<Heap*>(resourceHeap), std::move(committedResource));
 
+        Microsoft::WRL::ComPtr<ID3D12Pageable> pageable;
```

Comment:
Remove? Doesn't seem to do anything.

Reply:
Ah ignore this for now. Made a mistake when rebasing.